torch_xla/csrc/tensor.cpp

```diff
@@ -284,6 +284,26 @@ void XLATensor::SetXlaData(torch::lazy::BackendDataPtr handle, bool sync) {
   }
 }
 
+void XLATensor::CreateShardedIrValue(const ShardingSpecPtr sharding_spec) {
+  torch::lazy::Value old_value = GetIrValue();
```

Comment:
Use `CurrentIrValue()` ?

Reply:
I was getting crashes with Current, it looks like ir_value is not always defined when we call this method.